common/OpTestUtil.py

```diff
@@ -2100,6 +2100,46 @@ def distro_name(self):
         else:
             return "unknown"
 
+    def get_distro_details(self):
+        '''
+        1. Reads the contents of /etc/os-release
+        2. Formulates the same into a dictionary format.
+        3. Returns this dictionary in the format as below:
+        {
+            'NAME': ['"Red Hat Enterprise Linux"'], 
+            'VERSION': ['"8.8 (Ootpa)"'], 
+            'ID': ['"rhel"']
+            . . . .
+            . . . .
+        }
+
+        :param None
+        :return distro_details: contents in dictionary format
+        '''
+        temporary_list = []
+        distro_details = {}
+        host = self.conf.host()
+        res = host.host_run_command("cat /etc/os-release")
+
+        loop_count = 0
+        while loop_count < len(res):
+            temporary_list.append(res[loop_count].split("="))
```

Comment:
res contains list of lines ? and temporary_list contains multiple list ?

Reply:
> res contains list of lines ? and temporary_list contains multiple list ?

Yes, that's right!